### ISSUE TITLE
Segfaults

### ISSUE THREAD

**Issue by gribbs88** [jdolan/quake2#20]
I don't know if you are interested in fixing this. I like your quetoo q2, however it constantly segfaults over several linux distros. Currently running Kubuntu 18.04.2 LTS

Torment - by Pan
Julian entered the game              
anticheat Julian is not using anticheat.
usage: connect <server>
AlexandriaCorte was blown away by Optica's super shotgun
couldn't exec cfg/torment.cfg
[xcb] Unknown request in queue while dequeuing
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
quake2: ../../src/xcb_io.c:165: dequeue_pending_request: Assertion `!xcb_xlib_unknown_req_in_deq' failed.
Aborted (core dumped)
chips@chips-Precision-WorkStation-T3500:~/.quake2$ 

It ran well over two previous maps then boom. Has done it with Slackware, Debian, and Centos as well. 

Thanks,

**Comment by SpineyPete** [comment]
You're trying to use it to run Q2 games?
Quetoo is a stand-alone game, not a Quake 2 client, common misunderstanding apparently.

**Comment by gribbs88** [comment]
No, sorry. I mean the Quake 2 client for Linux. Not Quetoo. It's segfaulting consistently unfortunately.

**Comment by jdolan** [comment]
While I would absolutely welcome a patch that addresses whatever this crash actually is, I don't currently have plans to work on this fork anymore. Quetoo takes up my Quake hacking time these days, which has become quite scarce.

**Comment by gribbs88** [comment]
Ok, thanks for replying.

**Comment by rgosens2** [comment]
Also occasionally segfaults on macOS.
